clarity_ext/context.py

```diff
@@ -196,6 +196,104 @@ def commit(self):
             obj.commit()
 
 
+class FakingEntityMonkey:
+    """
+    Prepare genologics.entities for an entity based mock by
+    overwriting methods to use fake data instead.
+    NOTE! This monkey is not complete, right now it's just adapted for the
+    dilution script. It has to be updated for future extended testing.
+    Process.input_output_maps is not yet overwritten.
+    """
+
+    def __init__(self):
+
+        self._overwrite_entity()
+        self._overwrite_string_descriptor()
+        self._overwrite_process()
+        self._overwrite_lims()
+        self._overwrite_entity_list_descriptor()
+
+    def reset(self):
+        EntityListDescriptor.__get__ = self.entity_list_descriptor_get
```

Comment:
We talked about the approach of using mock objects defined only in the test itself instead, i.e. one abstraction layer closer, so to speak. Those kinds of tests tend to be super fast and probably simpler to maintain. Are you not considering doing that anymore? If you do, and this is temporary, I would suggest adding a comment on that, and again, move it closer to the test so that it's not something one needs to read to get an overview of the project.